tests/full_wlm/test_consolidation_functions.py

```diff
@@ -0,0 +1,118 @@
+import os
```

Comment:
Not to be overly nitpicking, but can we use a more descriptive name for this file? Something more along the lines of `test_wlm_helper_functions.py`? I think `consolidation_functions` is defined within the scope of this PR, but not too much elsewhere.

Reply:
Not overly nitpicking at all! I was a little weary on the name myself. I like the name `test_wlm_helper_functions.py` as is so I changed it to that!!

Rename `test_consolidation_functions.py` -> `test_wlm_helper_functions.py` at 32e2378